tests/virtualnetwork/virtualnetworkDeploy_test.go

```diff
@@ -129,6 +129,47 @@ func TestDeployVirtualNetworkValidVnetPeering(t *testing.T) {
 	test.ApplyIdempotent().ErrorIsNil(t)
 }
 
+// TestDeployVirtualNetworkValidUniDirectionalVnetPeering tests the deployment of a virtual network
+// with unidirectional peering to a hub virtual network.
+func TestDeployVirtualNetworkValidUniDirectionalVnetPeering(t *testing.T) {
+	t.Parallel()
+
+	utils.PreCheckDeployTests(t)
+	testDir := "testdata/" + t.Name()
```

Comment:
```suggestion
	testDir := ""
```

Reply:
@jaredfholgate please can you fix this to enable the deployment tests to run?

The test dir is inly required when calling the module from another root module.